.eslintrc.js

```diff
@@ -23,5 +23,11 @@ module.exports = {
   },
   overrides: [
     ...baseline.overrides,
+    {
+      files: ['contributor-dashboard-legacy/**'],
+      rules: {
+        'import/no-unresolved': 'off', // TODO, to fix at one point
```

Comment:
This is because we are not using the pnpm workspace for contributor-dashboard-legacy. 

One thing that I didn't try is to simply add `contributor-dashboard-legacy` in the pnpm workspace. Maybe one day in the future.